medicines/pars-upload/src/auth/authPopup.js

```diff
@@ -1,20 +1,46 @@
 import { UserAgentApplication } from 'msal'
 import { msalConfig, loginRequest } from './authConfig'
 
-export function getAccount() {
-  const myMSALObj = new UserAgentApplication(msalConfig)
-  const account = myMSALObj.getAccount()
+export async function getAccount() {
+  const msalInstance = new UserAgentApplication(msalConfig)
+  const account = msalInstance.getAccount()
 
   if (account) {
+    // TODO:
+    //
+    // Temporary hack whilst we wait for implicit granting of ID tokens to be
```

Comment:
To me this doesn't make sense, we have the token, why is it a hack to use it?

We do almost the same as in the documentation below apart from not using the `silent` approach